frontend/src/components/Workspace/FlowChart/FlowChartNode/NodeContainer.tsx

```diff
@@ -35,7 +35,7 @@ export const NodeContainer: FC<{
         flexDirection: "column",
       }}
     >
-      <Box margin={1}>{children}</Box>
+      <Box marginX={1}>{children}</Box>
```

Comment:
@ReiHashimoto I only used marginX because to avoid the content from overflowing, it also affects the status and close button, so I increased the height and used margin, pls check it again

Reply:
@ReiHashimoto I only used marginX because to avoid the content from overflowing, it also affects the status and close button, so I increased the height and used margin, pls check it again